Core/MoneyManager.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Core
+{
+    public static class MoneyManager
+    {
+        public static Money Convert(this Money money, Currency currency)
+        {
+            if (money.Currency == currency)
+            {
+                return money;
+            }
+            throw new ArgumentException("Invalid Currency");
+        }
+
+        public static Currency ParseCurrency(string currency)
+        {
+            switch (currency.ToLowerInvariant())
```

Comment:
Codacy found an issue: [Change this normalization to 'ToUpperInvariant()'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5924375)